core/main.go

```diff
@@ -448,8 +448,8 @@ func (t *Textile) UpdateCh() <-chan Update {
 }
 
 // ThreadUpdateCh returns the thread update channel
-func (t *Textile) ThreadUpdateCh() <-chan ThreadUpdate {
-	return t.threadUpdates
+func (t *Textile) ThreadUpdateCh() *Listener {
```

Comment:
this now returns a listener. should the method be renamed to reflect this change?

Reply:
sounds right... `GetThreadUpdateListener`?